test/app-tap/datetime.test.lua

```diff
@@ -5,6 +5,12 @@ local test = tap.test('errno')
 local date = require('datetime')
 local ffi = require('ffi')
 
+--[[
+    Workaround for #6599 where we may randomly fail on AWS Graviton machines,
+    while it's working properly when jit disabled.
+--]]
+if jit.arch == 'arm64' then jit.off() end
```

Comment:
I would be okay with just `jit.off()` without filtering by architecture, but if we do the filtering, I would prefer to re-verify it on amd64 in several ways (as I asked in the issue).

Reply:
Once, again, I

> I would prefer to re-verify it on amd64 in several ways (as I asked in the issue).

I've verified it thousand times. Once again - x64 works with GC64 and without. That's clearly ARM64/JIT specific.